docker-entrypoint.sh

```diff
@@ -24,4 +24,9 @@ fi
 # Run Data Migrations Only (Static files can remain local)
 python ./joplin/manage.py migrate --noinput
 
+if [ "$LOAD_DATA" == "on" ]; then
```

Comment:
Instead of loading data from fixtures files, or using a broken migration script with hardcoded data, we can simply load the latest backup (assuming it was created using the current migrations)